src/solver/utils/ortools_utils.cpp

```diff
@@ -354,11 +352,9 @@ extern "C"
                                    double* bMin,
                                    double* bMax,
                                    int* typeVar,
-                                   int nbVar,
-                                   PROBLEME_SIMPLEXE* Probleme)
+                                   int nbVar)
     {
-        Probleme->TypeDeVariable = typeVar;
-        change_MPSolver_variables_bounds(solver, bMin, bMax, nbVar, Probleme);
+        change_MPSolver_variables_bounds(solver, bMin, bMax, typeVar, nbVar);
```

Comment:
Remove function `change_MPSolver_variables_bounds` and place its implementation here ?

Reply:
Done